internal/otelcollector/config/metric/agent/testdata/config_istio_not_active.yaml

```diff
@@ -11,6 +11,7 @@ service:
                 - resource/delete-service-name
                 - resource/insert-input-source-istio
                 - batch
+                - filter/drop-istio-metrics-to-internal-components
```

Comment:
The batch processor should be the last one. Let's make this one the first after the memory_limiter so that we do not perform other actions on metrics that will be dropped.